frontend/libs/studio-icons/src/react/icons/AccordionIcon.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import type { IconProps } from '../types';
 import { SvgTemplate } from './SvgTemplate';
 
-export const Accordion = (props: IconProps): JSX.Element => {
+export const AccordionIcon = (props: IconProps): JSX.Element => {
```

Comment:
As we are already in the process of changing the method name, why not consider updating the return type of the methods too? The current use of `JSX.Element` is deprecated, and it is recommended to replace it with `React.ReactElement`. 😊

Reply:
Hi @framitdavid. The return types of methods have been updated